cd/prow/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Container image for running ACK CD prow jobs.
```

Comment:
So ideally I'd like to take this container and re-use it for integration tests, since it seems to have all the necessary tools (buildah, git, awscli). I think this might serve as the base image for the integration test Dockerfile, as part of a multi-stage build.

Reply:
I will update the description.